apps/omg_child_chain/lib/api.ex

```diff
@@ -36,13 +37,16 @@ defmodule OMG.ChildChain do
           {:ok, %{txhash: Transaction.tx_hash(), blknum: pos_integer, txindex: non_neg_integer}}
           | {:error, submit_error()}
   def submit(transaction) do
+    Tracer.start_span("transaction")
```

Comment:
:+1: on the telemetry, but same comments as before - maybe let's just test out the results in a live env and then polish out the implementation.

Having said that, it seems it would be possible to just swap out the implementation of our `@decorate measure_event()` and `decorate measure_that_other_thing` which were implemented in the previous metrics effort by @pthomalla.

I think transfering all the metrics captured currently to spandex could be a two liner modification using https://github.com/spandex-project/spandex#decorators

I think `decorate trace` becomes an AppSignal "transaction" and `decorate span` becomes an AppSignal "event", am I parsing the spandex' README correctly?